bin.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+'use strict'
+
+/** @type {typeof import("./source/bin.js") } */
```

Comment:
this should automatically be changed to `bin.ts` from `bin.js` once boundation runs on it again